src/_guides/libraries/testing.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: default
+title: "Dart Testing"
+description: "How to test Dart, Flutter, and 'Dart for the Web' applications?"
+---
+
+Software testing, an important part of app development, helps verify that
+your app is working correctly before unleashing it on your users.
+This Dart testing guide outlines several types of testing, and points
+you to where you can learn more about testing on the Flutter, Dart VM,
```

Comment:
Instead of:
> where you can learn more about testing on the Flutter, Dart VM, and 'Dart for the Web' platforms.

How about:
> where you can learn how to test your [mobile](https://flutter.io/), [web](https://webdev.dartlang.org/), and [server-side apps and scripts](https://sz-www-1.firebaseapp.com/dart-vm).

(We can also drop the "and scripts" w/o too much loss.)